signin.js

```diff
@@ -13,6 +13,10 @@ const checkEmailErrorMessage =  '이메일을 확인해주세요.';
 const checkPasswordErrorMessage = '비밀번호를 확인해주세요.';
 const iconEye = document.querySelector('.password-eye')
 
+if(localStorage.getItem("accessToken") !== null) {
```

Comment:
아래와 같이 더 간단하게 작성할 수 있을 것 같아요~! 
```suggestion
if(localStorage.getItem("accessToken"))
```